spec/lib/vanagon/platform/windows_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'vanagon/platform'
+load File.expand_path(File.join(File.dirname(__FILE__), "../../../..", "lib", "vanagon.rb"))
```

Comment:
To work out what VANAGON_ROOT is - See: https://github.com/puppetlabs/vanagon/blob/master/bin/build#L2